### DISCUSSION TITLE
[sdcard] The false warning message "The app doesn't have the permission to save this file, all changes will be lost" should not be displayed

### DISCUSSION THREAD

**Discussion by GregoryTrzonkowski** [gsantner/markor#1605] [Q&A] [answered]
## General information
* App version: 2.8.0
* Android version: 11
* Device: Xiaomi Redmi Note 8t

### Description
Steps: 
1. open Markor
2. Browse to SD card location
3. Press "plus" button
4. type name of the note file
5. tap "OK"

Actual result:
The warning message is shown: "Warning! The app doesn't have the permission to save this file, all changes will be lost. There is no 
 possibility to resolve the problem or to confirm something, Android denies any access here." User can input text, save the file and later open it successfully.

Expected result:
The warning message is not shown.

Screenshot of the warning message:
![markor warning message](https://user-images.githubusercontent.com/41368637/152401376-9e841ece-5f4b-4e90-b0b4-c70d301f3146.jpg)

**Comment by schnapphans** [comment]
I had the same problem.  See my solution here #1603

**Comment by gsantner** [comment, marked as answer]
I made some improvements for SD-Card support recently. Please see <https://github.com/gsantner/markor/issues/603>.
